contents/newsletter/first-time-founders.md

```diff
@@ -0,0 +1,133 @@
+---
+title: How first-time founders fail
+date: 2024-04-10
+author:
+  - andy-vandervell
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/newsletter/beyond-10x-engineer/super-hog.png
+featuredImageType: full
+tags:
+  - Being a founder
+  - Product-market fit
+crosspost:
+  - Founders
+  - Blog
+---
+
+Read the footnote if "trust me bro" isn't enough for you, but statistically:
+
+1. First-time founders fail more often than everyone else.
+2. Founders who previously failed are only _marginally_ more successful – like a percentage point or two.
+3. Previously successful founders are _way_ more likely to succeed than either of the above.
+
+Therefore, if you want to learn how first-time founders fail and avoid their mistakes, it's best to ask the ones who found success after their first startup(s) failed. 
+
+So, I did. Here's what they said.
+
+## Paul Copplestone (Co-founder of Supabase)
+
+- **ServisHero:** 2015 to 2017 – left but company still active
+- **Nimbus for Work:** 2018 to 2019
+- **Supabase:** 2020 to present
+
+THIS WILL BE A GRAPHIC
+
+Supabase is lean, product-led, and grows by word-of-mouth. It's the polar opposite of Paul's first startup, a marketplace for finding local services in South East Asia.
+
+They [raised millions](https://techcrunch.com/2016/03/20/servishero-a-mobile-app-for-finding-local-services-in-southeast-asia-lands-2-7m/), generated early buzz, and blitzscaled their way to double-digit growth by expanding into new countries.
+
+But Paul soon saw signs it wasn't working. Retention was poor. Growth was coming from marketing spend, rather than word-of-mouth. Worst of all, they were over hiring and competing with big tech on salaries – what Paul refers to as "playing starting".
```

Comment:
```suggestion
Paul soon saw signs it wasn't working. Retention was poor. Growth was coming from marketing spend, rather than word-of-mouth. Worst of all, his company was over-hiring and competing with big tech on salaries – what Paul refers to as "playing starting."
```